<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Simple Subtitle Editor</string>
  </property>
  <property name="windowIcon">
   <iconset resource="myresources.qrc">
    <normaloff>:/png/png/se.png</normaloff>:/png/png/se.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableWidget" name="table">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>761</width>
      <height>191</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="columnCount">
     <number>5</number>
    </property>
    <column>
     <property name="text">
      <string>#</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Start time</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>End time</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Duration</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Text</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>481</width>
      <height>41</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="text">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Text:</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>230</y>
      <width>261</width>
      <height>91</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Opened file info:&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Titles duration - &lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Format - &lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Number of titles - &lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;File size in bytes - &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteItems">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete row(s)</string>
    </property>
   </widget>
   <widget class="QLabel" name="currentInfo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>761</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="insertItem">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>340</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Insert row</string>
    </property>
   </widget>
   <widget class="QTimeEdit" name="starttimeEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>118</width>
      <height>22</height>
     </rect>
    </property>
    <property name="showGroupSeparator" stdset="0">
     <bool>false</bool>
    </property>
    <property name="maximumTime">
     <time>
      <hour>9</hour>
      <minute>59</minute>
      <second>59</second>
     </time>
    </property>
    <property name="currentSection">
     <enum>QDateTimeEdit::HourSection</enum>
    </property>
    <property name="displayFormat">
     <string>HH:mm:ss.zzz</string>
    </property>
    <property name="currentSectionIndex">
     <number>0</number>
    </property>
    <property name="time">
     <time>
      <hour>0</hour>
      <minute>0</minute>
      <second>0</second>
     </time>
    </property>
   </widget>
   <widget class="QTimeEdit" name="endtimeEdit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>290</y>
      <width>118</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximumTime">
     <time>
      <hour>9</hour>
      <minute>59</minute>
      <second>59</second>
     </time>
    </property>
    <property name="currentSection">
     <enum>QDateTimeEdit::HourSection</enum>
    </property>
    <property name="displayFormat">
     <string>HH:mm:ss.zzz</string>
    </property>
    <property name="calendarPopup">
     <bool>false</bool>
    </property>
    <property name="currentSectionIndex">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="text_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Start time:</string>
    </property>
   </widget>
   <widget class="QLabel" name="text_3">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>270</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>End time:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="changeTime">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>290</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Change time</string>
    </property>
   </widget>
   <widget class="QLabel" name="timeErrors">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>400</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>230</y>
      <width>20</width>
      <height>161</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>761</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="rowLabel">
    <property name="geometry">
     <rect>
      <x>746</x>
      <y>210</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="ranges">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>206</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="mergeButton">
    <property name="geometry">
     <rect>
      <x>626</x>
      <y>340</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Merge two</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="applyToAll">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>340</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Apply to all</string>
    </property>
   </widget>
   <widget class="QPushButton" name="splitButton">
    <property name="geometry">
     <rect>
      <x>626</x>
      <y>370</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Split to more</string>
    </property>
   </widget>
   <widget class="QPushButton" name="moveButton">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>340</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Move in time</string>
    </property>
   </widget>
   <widget class="QPushButton" name="distanceButton">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>370</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Change distance</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resolveButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>370</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Resolve errors</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionInsert_row"/>
    <addaction name="actionDelete_row_s"/>
    <addaction name="actionChange_FPS"/>
    <addaction name="actionMerge_two"/>
    <addaction name="actionSplit_to_more"/>
    <addaction name="actionMove_in_time"/>
    <addaction name="actionChange_distance"/>
    <addaction name="actionResolve_errors"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="myresources.qrc">
     <normaloff>:/png/png/open.png</normaloff>:/png/png/open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open File</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="myresources.qrc">
     <normaloff>:/png/png/x.png</normaloff>:/png/png/x.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="myresources.qrc">
     <normaloff>:/png/png/Save-icon.png</normaloff>:/png/png/Save-icon.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionInsert_row">
   <property name="text">
    <string>Insert row</string>
   </property>
  </action>
  <action name="actionDelete_row_s">
   <property name="text">
    <string>Delete row(s)</string>
   </property>
  </action>
  <action name="actionChange_FPS">
   <property name="text">
    <string>Change FPS</string>
   </property>
  </action>
  <action name="actionMerge_two">
   <property name="text">
    <string>Merge two</string>
   </property>
  </action>
  <action name="actionSplit_to_more">
   <property name="text">
    <string>Split to more</string>
   </property>
  </action>
  <action name="actionMove_in_time">
   <property name="text">
    <string>Move in time</string>
   </property>
  </action>
  <action name="actionChange_distance">
   <property name="text">
    <string>Change distance</string>
   </property>
  </action>
  <action name="actionResolve_errors">
   <property name="text">
    <string>Resolve errors</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="myresources.qrc"/>
 </resources>
 <connections/>
</ui>
